Core/Core/Conversations/APIConversation.swift

```diff
@@ -248,3 +248,23 @@ public struct PostAddMessageRequest: APIRequestable {
     public var path: String { "conversations/\(conversationID)/add_message" }
     public let method = APIMethod.post
 }
+
+public struct PostConversationRequest: APIRequestable {
+    public typealias Response = [APIConversation]
```

Comment:
I'm surprised that the response is an array.

Reply:
Yep that threw me off at first as well.

Reply:
I can add some comments for this.